app/main.py

```diff
@@ -2,0 +1,19 @@
+import os
+
+
+def move_file(string: str) -> None:
+    words = string.split()
+    old_file_name = words[1]
+    destination_path = os.path.abspath(words[-1])
+
+    new_dir = os.path.dirname(destination_path)
```

Comment:
Use `os.path.join` to concatenate parts of the path for cross-platform compatibility, as recommended in the checklist.